internal/api/scim/integration_test/users_update_test.go

```diff
@@ -0,0 +1,284 @@
+//go:build integration
+
+package integration_test
+
+import (
+	"context"
+	_ "embed"
+	"fmt"
+	"net/http"
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/brianvoe/gofakeit/v6"
+	"github.com/muhlemmer/gu"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/text/language"
+
+	"github.com/zitadel/zitadel/internal/api/scim/resources"
+	"github.com/zitadel/zitadel/internal/api/scim/schemas"
+	"github.com/zitadel/zitadel/internal/integration"
+	"github.com/zitadel/zitadel/internal/integration/scim"
+	"github.com/zitadel/zitadel/internal/test"
+	"github.com/zitadel/zitadel/pkg/grpc/user/v2"
+)
+
+var (
+	//go:embed testdata/users_update_test_full.json
+	fullUserUpdateJson []byte
+
+	minimalUserUpdateJson []byte = simpleReplacePatchBody("nickname", "foo")
+
+	// remove comments in the json, as the default golang json unmarshaler cannot handle them
+	// the test file is much easier to maintain with comments
+	removeCommentsRegex = regexp.MustCompile("(?s)//.*?\n|/\\*.*?\\*/")
+)
+
+func init() {
+	fullUserUpdateJson = removeComments(fullUserUpdateJson)
+}
+
+func removeComments(json []byte) []byte {
+	return removeCommentsRegex.ReplaceAll(json, nil)
+}
+
+func TestUpdateUser(t *testing.T) {
+	fullUserCreated, err := Instance.Client.SCIM.Users.Create(CTX, Instance.DefaultOrg.Id, fullUserJson)
+	require.NoError(t, err)
+
+	defer func() {
+		_, err = Instance.Client.UserV2.DeleteUser(CTX, &user.DeleteUserRequest{UserId: fullUserCreated.ID})
+		require.NoError(t, err)
+	}()
+
+	// secondary organization with same set of users,
+	// these should never be modified.
+	// This allows testing list requests without filters.
+	iamOwnerCtx := Instance.WithAuthorization(CTX, integration.UserTypeIAMOwner)
```

Comment:
If you want isolated instances to separate the testing, you could use `integration.NewInstance` per test.